modules/ROOT/pages/get-started-verify-install.adoc

```diff
@@ -429,7 +429,7 @@ You should see the following response:
     "rev": "4-03f1ba127340e8c50c31a36279298e60" // <.>
 }
 ----
-<.> You can see here that the delete counts as the fourth change/revision. Again "ok":true indicates the delete was successful.
+<.> You can see here that the delete counts as the fourth change/revision. Again `"ok":true` indicates the delete was successful.
```

Comment:
The delete counts as the fourth change/revision. `"ok":true` indicates the delete was successful.